theano/tensor/basic.py

```diff
@@ -968,6 +968,53 @@ def _pack(x):
         return [x]
 
 
+def check_and_normalize_axes(x, axis):
+    """
+    Check axes, normalize and convert them to a Python list of integers.
+    Return an empty list if argument is None.
+
+    Parameters
+    ----------
+    x: Tensor variable
+    axis = Integer, tuple or list of integers
+
+    Returns
+    -------
+    axis: list of integers
+    """
+    x = as_tensor_variable(x)
```

Comment:
It is standard to call as_tensor_variable on exising Theano variable. This will be a no op and just return the input. It prevent making if at many places in Theano. So this is good.